franka_gazebo/config/sim_controllers.yaml

```diff
@@ -1,3 +1,51 @@
+position_joint_trajectory_controller:
+  type: position_controllers/JointTrajectoryController
+  joints:
+    - panda_joint1
+    - panda_joint2
+    - panda_joint3
+    - panda_joint4
+    - panda_joint5
+    - panda_joint6
+    - panda_joint7
+  constraints:
+    goal_time: 0.5
+    panda_joint1: { goal: 0.05}
```

Comment:
Here I used the traj goal tolerance that is found in the `franka_control` package. I however think this is too big `https://github.com/frankaemika/franka_ros/issues/201`. Please update the `franka_control` package so that I can update it here.